### PR TITLE
User onboarding

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Displays time, and current weather in sync
Wallpaper and quote change on every refresh
Added main focus for the day

image

image

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -1,25 +1,10 @@
...
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
-}
+	"manifest_version": "1",
```
Comment (Rohitprasad83) [inline public/manifest.json:2]: is manifest version 1 working on chrome?

File: src/contexts/browser-context.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { createContext, useContext, useReducer } from "react";
+import { browserReducer } from "./browserReducer";
+
+const BrowserContext = createContext();
+
+const BrowserProvider = ({ children }) => {
+    const [browserState, browserDispatch] = useReducer(browserReducer, {
```
Comment (prakashsakari) [inline src/contexts/browser-context.js:7]: you can destructure state values directly over here, so there is no need to destructure it every where

File: src/contexts/browserReducer.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const browserReducer = (state, action) => {
```
Comment (prakashsakari) [inline src/contexts/browserReducer.js:1]: you can destructure type and payload, over here so that you can directly use payload without the need of action

**Review by Rohitprasad83 — APPROVED** [review]
Clean code and good use of useReducer

**Review by prakashsakari — APPROVED** [review]
Code, Ui and folder structure are good
Added few comments you can reflect on
Overall good to merege

File: public/manifest.json (hunk 1 above)
Comment (ranitasahaa1230) [reply public/manifest.json:2]: Nope its not working

File: src/contexts/browser-context.js (hunk 2 above)
Comment (ranitasahaa1230) [reply src/contexts/browser-context.js:7]: will modify it

File: src/contexts/browserReducer.js (hunk 3 above)
Comment (ranitasahaa1230) [reply src/contexts/browserReducer.js:1]: Cool, will update it too

**Pull request merged**